sdk/nodejs/tests/runtime/jsClosureCases_8.js

```diff
@@ -8,7 +8,7 @@ cases.push({
     func: async function (a) { await a; },
     expectText: `exports.handler = __f0;
 
-function __f0() {
+function __f0(__0) {
```

Comment:
note: we never actually consume these.  i didn't want to change semantics in some unforseen manner.  so in the body, we still reference `arguments`.  These params just serve to ensure that `func.length` returns the *declared* param count so that we work with other libraries that check for that.